requests_toolbelt/utils/formdata.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""Implementation of nested form-data encoding function(s)."""
+from .._compat import basestring
+from .._compat import urlencode as _urlencode
+
+
+__all__ = ('urlencode',)
+
+
+def urlencode(query, doseq=False, safe='', encoding=None, errors=None):
```

Comment:
`*args` and `**kwargs` don't lend themselves to discoverability but I understand your point. I may as well just do that.